### ISSUE TITLE
val_loss is not available during training

### ISSUE THREAD

**Issue by YuvalSilman** [keras-team/keras#13968]
When using fit on my model i receive the error: 
"Can save best model only with val_loss available, skipping."
My callbacks are using val_loss monitoring.
After some investigation i found out that the val loss is being calculated and it is accessable after the training was finished but during training the callbacks cant reach it for some reason. 
Has anyone encountered this issue before? can someone help me?

My code:
![image](https://user-images.githubusercontent.com/49240261/79211909-83faf680-7e4f-11ea-86c8-11f1f46875ad.png)
![image](https://user-images.githubusercontent.com/49240261/79211979-9bd27a80-7e4f-11ea-8e66-981447324b45.png)

**Comment by kumariko** [comment]
@YuvalSilman We are checking to see if you still need help on this issue. We recommend that you upgrade to 2.6 which is latest stable version of TF and let us know if the issue still persists in newer versions. please provide a code snippet in comment to reproduce the issue reported here. we will get you the right help.Thanks!
